seed/static/seed/partials/derived_columns_editor.html

```diff
@@ -0,0 +1,142 @@
+                        class="form-control"
+                        aria-label="Parameter name"
+                    >
+                    <div ng-if="parameter.errors.invalid_parameter_name" class="text-danger">Parameter name is invalid.</div>
+                    <div ng-if="parameter.errors.duplicate_parameter_name" class="text-danger">Parameter name "{$ parameter.parameter_name $}" is duplicated. Please change or remove one of the duplicate parameters.</div>
+                    <div ng-if="parameter.errors.expression_missing_parameter" class="text-danger">Parameter is not used in the expression. Use <code>${$ parameter.parameter_name $}</code> to reference the parameter.</div>
+                </div>
+                <div>
+                    <input
+                        type="text"
+                        ng-model="parameter.source_column"
+                        ng-blur="updated_parameter_or_expression()"
+                        class="form-control"
+                        placeholder="Source column name"
+                        uib-typeahead="column as column.displayName for column in (derived_column.inventory_type === 'Property' ? property_columns : taxlot_columns) | filter:{displayName:$viewValue}"
+                        typeahead-template-url="typeahead.html"
+                        typeahead-editable="false"
+                        typeahead-show-hint="true"
+                        typeahead-min-length="0"
+                        typeahead-select-on-exact="true"
+                        typeahead-select-on-blur="true"
+                        autocomplete="off"
+                        autocorrect="off"
+                        autocapitalize="off"
+                        spellcheck="false"
+                        aria-label="Source column name"
+                    >
+                    <div ng-if="parameter.errors.invalid_source_column" class="text-danger">Column name is invalid. Please select one of the provided choices.</div>
+                    <div ng-if="parameter.errors.duplicate_source_column" class="text-danger">Column is duplicated. Please change or remove one of the duplicate columns.</div>
+                </div>
+                <div ng-repeat-end>
+                    <button ng-if="parameters.length > 1" ng-click="delete_parameter($index)" class="btn btn-danger" aria-label="Delete">
+                        <span class="glyphicon glyphicon-remove"></span>
+                    </button>
+                </div>
+
+                <button ng-click="add_parameter()" class="btn btn-info">
+                    <span class="glyphicon glyphicon-plus"></span> Add parameter
+                </button>
+                <span></span>
+                <span></span>
+
+                <!-- expression -->
+                <h3>Expression</h3>
+                <span></span>
+                <span></span>
+                <div style="grid-column: 1 / 3">
+                    <input
```

Comment:
Probably a note for a future change: Related to creating an expression, it'd be great to see notes/instructions (maybe something that's collapsible?) showing the different operations and the syntax for each of them .

Reply:
I totally agree, will add a collapsable help section later 👍